### PR TITLE
Beo 13785 rename apiversion

### PR DESCRIPTION
**Pull request by shirbruchim**

### CODE REVIEW

**Comment by zoharbabin** [comment]
@shirbruchim did you confirm it works with the console code and that nothing breaks when this new param name is used instead of `version`?

**Comment by shirbruchim** [comment]
> @shirbruchim did you confirm it works with the console code and that nothing breaks when this new param name is used instead of `version`?

@zoharbabin  first need to merge this fix https://github.com/kaltura/xml-api-schema-openapi-converter/pull/29 which update openapi files with their latest version + fix an issue that was with ott generation - after that you can see that there is no change except "version" to "apiVersion"

**Comment by zoharbabin** [comment]
#29 was merged. regarding this pull - did you confirm that the OVP gen will not break because you changed version to apiVersion? you need to run both generators and see that neither break, not just ott gen

**Comment by shirbruchim** [comment]
@zoharbabin i run them both ott and ovp of course and you can see that only ott file was changed :)

File: lib/swagger-builder.js [hunk 1]
```diff
@@ -314,7 +314,7 @@ class SwaggerBuilder {
         type: 'object',
```
Comment (shirbruchim) [inline lib/swagger-builder.js:314]: @zoharbabin  any way you can see here that version property is relevant ONLY for ott target

Comment (zoharbabin) [reply lib/swagger-builder.js:314]: thanks for confirming. merged.

**Pull request merged**